core/services/ocr2/plugins/ocr2keeper/evm21/registry_check_pipeline.go

```diff
@@ -243,12 +253,16 @@ func (r *EvmRegistry) checkUpkeeps(ctx context.Context, payloads []ocr2keepers.U
 		index := indices[i]
 		if req.Error != nil {
 			latestBlock := r.bs.latestBlock.Load()
+			latestBlockNumber := int64(0)
```

Comment:
🏄 I would swap lines 255 and 256 so the nil check immediately follows the declaration of what the nil check examines